_posts/2021-05-24-pulpcore-3.13-is-generally-available.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Pulp 3.13 is now Generally Available
+author: Melanie Corr
+tags:
+  - release
+---
+As well as a myriad of bugfixes and documentation improvements, this release contains some notable new features and enhancements. Let's take a look at the headline items of this release.
+
+## Control how many repo versions you retain [#8368](https://pulp.plan.io/issues/8368)
+
+Repository versioning was one of the main features introduced with Pulp 3. Every time you add or remove content from a repository, a new version is created. While this feature has its advantages, you might want to set some constraints to reduce the cleanup and maintenance job of retaining an a large volume of repository versions in your database.
+
+With this release, you can now set limits to the number of repository versions you retain. You can apply this on a per-repository basis, so that you can still retain as many versions of one repository while constraining the number of versions that are kept for another.
+
+Here is a demo of how you can specify the number of versions to retain. This demo uses our lovely new [Pulp 3 CLI](https://github.com/pulp/pulp-cli)!
+
+<a href="https://asciinema.org/a/412393" target="_blank"><img src="https://asciinema.org/a/412393.svg" /> </a>
+
+## New repository version and publication views [#4832](https://pulp.plan.io/issues/4832)
```

Comment:
```suggestion
## New repository version and publication endpoints [#4832](https://pulp.plan.io/issues/4832)
```

Reply:
Everything's an endpoint :) 
I used to work on IBM Endpoint Manager. 
It's as clear as service, but I accept !